src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,354 @@
+- Jake Balaš
+- Konstantin Ignatov
+- Kshitijaa Jaglan
+- Marcin Wierzbanowski
+- Mia Bajić
+- Oleh Kostromin
+- Raquel Dou
+
+## Financial Aid
+
+Speakers or attendees who need financial assistance to attend can apply for support. This team manages financial aid applications, selects recipients, handles communications, and ensures invoices are submitted:
+
+- Vicky Twomey-Lee
+- Doreen Peace Nangira Wanyama
+- Koti Vellanki
+- Shekhar Koirala
+- Artur Czepiel
+
+
+## Operations
+This is the largest team, divided into several smaller teams. You can meet them onsite or when contacting our helpdesk, staffed weekly by rotating team members.
+
+Operations Team:
+
+- Aneta Popelová
+- Anežka Müller
+- Aris Nivorlis
+- George Margaritis
+- George Zisopoulos
+- Jakub Červinka
+- Jake Balaš
+- John Robert
+- Kateřina Vaňková
+- Liberta Gani
+- Mahe Iram Khan
+- Martin Borus
+- Moisés Guimarães
+- Niklas Mertsch
+- Piotr Gnus
+- Saurav Pandey
+- Simon Peter Dagbui
+
+
+## Discord Team
+
+If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organisers. This team manages the Discord server, ensuring only participants have access and proper permissions.
+
+Huge thanks to **Niklas Mertsch** for leading Discord and bot management!
+
+The work on maintaining and updating the Discord bot can be found on the [EuroPython Discord bot repo](https://github.com/EuroPython/discord).
+
+
+## A/V Team
+Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.
+
+Big thanks to **Raquel Dou**,  **Piotr Gnus** and **Anežka Müller** for A/V coordination.
+
+
+## Onsite Volunteers Team
+
+Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.
+
+- xxx
+
+
+# Events & Workshops Organisers
+Beyond talks and tutorials, EuroPython includes engaging events and workshops.
+
+## Django Girls
+A workshop designed to inspire less represented groups, particularly women, to get interested in technology. Big thanks to Doreen Peace Nangira Wanyama for organizing!
+
+
+## Humble Data
+
+Data workshop designed for total beginners coming from less represented groups. Organisers and mentors:
+
+- Cheuk Ting Ho
+- Jodie Burchell
+- Martin Borus
+- Sena Şahin
+
+Special thanks to **Jodie Burchell** for coordinating beginner workshops.
+
+## Unconference Day
+In parallel with sprints and workshops, there was an unconference day featuring Q&A, discussions, and unconference-style sessions, inspired by the same concept hosted at PyCon CZ 23. Huge thanks to organisers **Daria Grudzien Linhart** and **Honza Javorek**.
+
+## PyLadies Events
+On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria José Molina** for coordination.
+## Community booths
```

Comment:
There was also the community organiser [open space](https://ep2025.europython.eu/community-activities/#community-organisers-open-space). I know that @marinamorolopez @thepetk and @cmaureir were involved (and they would know who else helped). I wonder if it might be nice to have one heading for "Community events", then "Community Organisers Open Space" and "Community Booths" as its subheadings.

Reply:
suggestion added, check it out https://github.com/EuroPython/website/pull/1479#discussion_r2277031716